azure-pipelines.yml

```diff
@@ -71,7 +71,7 @@ jobs:
   steps:
   - task: UsePythonVersion@0
     inputs:
-      versionSpec: '3.7'
+      versionSpec: '3.8'

```

Comment:
Why previous job has 3.7 version and that has 3.8? Why not to align version to one value?

Reply:
I updated. You need to make a rebase